### DISCUSSION TITLE
Adding default Value inside field definition

### DISCUSSION THREAD

**Discussion by Archer-Thane** [amplication/amplication#2540] [Ideas]
That would be nice if the default value for the fields can be defined at the first place. for example, if there is a cart entity and it has a totalItems field (which is a whole number) then default value of zero can show that this Cart is empty.
That was the Idea.
Thanks for the great job on this project

**Comment by yuval-hazaz** [comment]
Thanks
See this feature request which is still open #1045 
We will be able to include it in a future version, no timeline yet
